docs/config/lua/keyassignment/SetWindowLevel.md

```diff
@@ -0,0 +1,30 @@
+# `SetWindowLevel`
+
+{{since('nightly')}}
+Set window level sepcified by the argument value. eg: `AlwaysOnTop` keeps the current window on top of other windows.
+
```

Comment:
Please update the docs to indicate that this is currently only implemented on macOS, and that these new assignments have no effect on other operating systems.

Reply:
same applies to the others right?